### PR TITLE
#3631 Fix broken deep links

### LINKED ISSUE #3631: Some Deep-Links are broken
- When accessing `/project/{project}/sequence/{shkeptncontext}/event/{eventId}`, `/event/{eventId}` is removed and the right stage is not selected.
- `project/{project}/service/{service}/context/{shkeptncontext}` does not select the last stage
- (optional) When a sequence/service is selected the latest stage isn't set in the URL. Without it refreshing/accessing would be inconsistent and could select a different stage.

### PR DESCRIPTION
**Pull request by ermin-muratovic**
Deep links should work again as expected and also units tests were introduced to check deep links and make sure the forwarding is working correctly

Closes #3631

### CODE REVIEW

File: bridge/client/app/_services/_mockData/evaluations-mock.ts [hunk 1]
```diff
@@ -0,0 +1,2760 @@
...
+      "service": "carts",
+      "stage": "staging",
+      "status": "succeeded"
+    },
+    "id": "843a4a6b-4dec-4328-8579-6af3d1a6c7cb",
+    "source": "lighthouse-service",
+    "specversion": "1.0",
+    "time": "2021-03-16T12:48:05.732Z",
+    "type": "sh.keptn.event.evaluation.finished",
+    "shkeptncontext": "6f1327d2-ded2-48ab-a1c6-e4f3d0ebe432",
```
Comment (jsoref) [inline bridge/client/app/_services/_mockData/evaluations-mock.ts:72]: Add this to `patterns.txt`:
```
# uuid:
[<({"'>][0-9a-fA-F]{8}-(?:[0-9a-fA-F]{4}-){3}[0-9a-fA-F]{12}[<'"})>]
```
From https://github.com/check-spelling/check-spelling/wiki/Configuration-Examples%3A-patterns

**Review by jsoref — CHANGES_REQUESTED** [review]
The other items should just be added to `expect.txt`, either by running the command suggested by the bot or manually.

File: bridge/client/app/_services/_mockData/evaluations-mock.ts (hunk 1 above)
Comment (ermin-muratovic) [reply bridge/client/app/_services/_mockData/evaluations-mock.ts:72]: I added the words to `expect.txt` and the pattern to `patterns.txt`
Thanks.

**Review by Kirdock — APPROVED** [review]
LGTM

**Pull request merged**